components/widgets/tag/__snapshots__/tag.test.tsx.snap

```diff
@@ -0,0 +1,89 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`components/widgets/tag/Tag should match the snapshot on show 1`] = `
```

Comment:
Snapshots when we shallow mount feel a bit incomplete to me.
I'd like to also assert on dom structure etc.
can we use mount for snapshot testing, since it is a "dump" component it won't be complex (3/5)

Reply:
well, i did not change the way this is tested. Would you mind opening a follow-up ticket? This could be a great HW candidate!